doc/Data Plane Code Generation/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -1,201 +1,146 @@
+#### 1. install dotnet template
+navigate to the sdk repo root director, and run the following commands:
 
-Open autorest.md in your `Azure.<group>.<service>` project. In the input-file definition, replace the file path with your swagger link, for example: [WebPubSub swagger link](https://github.com/Azure/azure-rest-api-specs/blob/39c7d63c21b9a29efe3907d9b949d1c77b021907/specification/webpubsub/data-plane/WebPubSub/stable/2021-10-01/webpubsub.json). Update a namespace definition to set the namespace C# generator will use when creating your types. Update the AAD token credential(for AAD token, refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template-dpg/Azure.Template.Generated/src/autorest.md#autorest-configuration) example) or Azure key credential (for Azure Key credential, refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/webpubsub/Azure.Messaging.WebPubSub/src/autorest.md#swagger-sources) example). Save your changes.
-
-Your `autorest.md` file should now look like:
-
-````md
-# `Azure.<group>.<service>` Code Generation
-
-Run `dotnet build /t:GenerateCode` to generate code.
-
-### AutoRest Configuration
-> see https://aka.ms/autorest
-
-``` yaml
-input-file:
-- <swagger-file-link>
-namespace: <namespace>
-public-clients: true
-data-plane: true
-security: AADToken
-security-scopes: <AAD-token-scope>
 ```
-````
-
-**Learn more:** see the [README.md](https://github.com/Azure/autorest.csharp/blob/feature/v3/readme.md) to learn more about Azure SDK Code Geneneration.
-
-#### 2. Run the Generate Command
-
-From a PowerShell command prompt, navigate to the directory holding `Azure.<group>.<service>.sln`. Run the following commands:
+dotnet new -i eng/templates/Azure.ServiceTemplate.Template
+```
 
+#### 2. dotnet new project
+- create project folder 'Azure.<group>.<service>'. e.g. Azure.IoT.DeviceUpdate under sdk/<service> folder
+- new project: navigate to the project folder, and run 'dotnet new' as following:
+  
 ```
-sdk\<your-service-name>\Azure.<group>.<service>\src> dotnet msbuild /t:GenerateCode
+sdk\<your-service-name>\Azure.<group>.<service>> dotnet new dataplane --libraryName [Client-Library-Title] --swagger [input-swagger-file-path] --securityScopes [security-scopes] --force
+```
+e.g.
+```
+dotnet new dataplane --libraryName DeviceUpdate --swagger https://github.com/dpokluda/azure-rest-api-specs/blob/be397aa65510bd4e8f87da539af2b0025f6f44ca/specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/preview/2020-09-01/deviceupdate.json --securityScopes https://api.adu.microsoft.com/.default --force
 ```
 
-#### 3. Explore the Generated Code
+#### 3. update the solution file if needed
+Run 'dotnet sln' as following to update the projects in the solution file:
+dotnet sln remove src\Azure.<group>.<service>.csproj
+dotnet sln add src\Azure.<group>.<service>.csproj
+dotnet sln remove tests\Azure.<group>.<service>.Tests.csproj
+dotnet sln add tests\Azure.<group>.<service>.Tests.csproj
```

Comment:
Add new lines. It is not adding `dotnet sln.....` in new lines. Check [here](https://github.com/Azure/azure-sdk-for-net/blob/bb0fbccfc33dd27d1ec6f0870022824d47181e61/doc/Data%20Plane%20Code%20Generation/AzureSDKCodeGeneration_DataPlane_Quickstart.md#3-update-the-solution-file-if-needed).